win32.mak

```diff
@@ -86,3 +86,28 @@ zip: detab tolf $(MAKEFILES)
 
 scp: detab tolf $(MAKEFILES)
 	$(SCP) $(SRCS) $(MAKEFILES) $(SCPDIR)
+
+
+################################################################################
+# Build and run tests
+################################################################################
+
+RDMD_TEST_COMPILERS = $(DMD)
+RDMD_TEST_EXECUTABLE = $(ROOT)\rdmd.exe
+RDMD_TEST_DEFAULT_COMPILER = $(DMD)
+
+VERBOSE_RDMD_TEST=0
+ifeq ($(VERBOSE_RDMD_TEST), 1)
+	override VERBOSE_RDMD_TEST_FLAGS:=-v
+endif
```

Comment:
I think this isn't supported by DigitalMars make

Reply:
Ho hum.  Probably best to just specify `VERBOSE_RDMD_TEST_FLAGS` directly then.

Reply:
Or just always pass `-v`, which is probably desirable.